trademapper/js/trademapper.yearslider.js

```diff
@@ -105,28 +164,17 @@ function($, d3) {
 		playPauseYearSlider: function() {
 			// if not enabled, do nothing
 			if (this.enabled === false) { return; }
-			// don't play if enable() hasn't been called
+
+			// if data hasn't been loaded, do nothing
 			if (this.minYear === 0) { return; }
```

Comment:
Haha, I know this isn't new code, and I'm not suggesting we change it, but it made me think that we can't use this tool to map trade of ancient Rome!